src/app/containers/MostRead/Canonical/LastUpdated.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { number, string, shape } from 'prop-types';
+import { scriptPropType } from '@bbc/gel-foundations/prop-types';
+import Timestamp from '@bbc/psammead-timestamp-container';
+import { shouldRenderLastUpdated } from '../utilities';
+
+const LastUpdated = ({ locale, prefix, script, service, timestamp }) =>
+  shouldRenderLastUpdated(timestamp) && (
+    <Timestamp
+      timestamp={timestamp}
+      dateTimeFormat="YYYY-MM-DD"
```

Comment:
Just checking - is this the desired date format?

Reply:
I think this is to compensate for languages where the date format is YYYY-MM-DD, the format is set to `LL` so that it should change depending on the service/locale. Rendering in `FrontPageMain` shows the news timestamps in the correct format:
![image](https://user-images.githubusercontent.com/53564281/73831451-48fc9700-47fe-11ea-9e5e-c75d51dbbe25.png)